build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.0'
+        classpath 'com.android.tools.build:gradle:3.2.0-alpha08'
```

Comment:
was this upgrade to an alpha dependency deliberate?

Reply:
Yes, Android Studio Preview 3.2 requested the change.